packages/ecs-gpu-init/ecs-gpu-init.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Initialize ECS GPU config
+# This has to be executed after the kernel modules are loaded
+# otherwise the userspace component of the driver will fail to
+# query the /dev devices
+After=load-kernel-modules.service
+# Block manual interactions with this service. It doesn't
+# make sense to regenerate the GPU config file if the ECS
+# agent won't read it when it changes
```

Comment:
If we expect the file to change at runtime (do we?) we could arrange to restart the ecs-agent service after it changes. If we don't then the comment is a bit confusing.

Reply:
We don't expect to change the file at runtime, but I thought it will be good to prevent users from attempting to change it. Can you elaborate on what part is confusing? :smile:

Reply:
The comment reads like a `TODO` to me, like we're saying there's a behavior change in the ECS agent that we want but we haven't addressed that yet.